packages/client-sdk-nodejs/src/config/storage-configuration.ts

```diff
@@ -1,6 +1,9 @@
-import {MomentoLoggerFactory} from '../';
+import {MomentoLoggerFactory, RetryStrategy} from '../';
 import {StorageTransportStrategy} from './transport/storage';
 
+// TODO: include ResponseDataReceivedTimeout as a top-level config?
+// Or keep at the StorageTransportStrategy level?
```

Comment:
This seems like lower level. We can keep it in transport strategy and add an accessor up here if it becomes an issue.

Reply:
oh whoops, I moved ResponseDataReceivedTimeout to be part of the specific RetryStrategy class it's used for. Will remove this comment